apps/crn-server/test/data-providers/contentful/users.data-provider.test.ts

```diff
@@ -781,6 +821,22 @@ describe('User data provider', () => {
           },
         });
       });
+      test('map tag value to a linked resource', async () => {
+        await userDataProvider.update('123', {
+          tags: [{ id: '1', name: '1' }],
```

Comment:
check out my comment related to the type definition for the `UserCreateObject`- you can see here that the `name` is being ignored